README.rst

```diff
@@ -89,11 +113,11 @@ In order to test the service is running, run::
 
 Top_
 
-Configuration
--------------
+SSH and PGP keys
+----------------
 
-You must to add the key files in the path /opt/fiware-aiakos/lib/public/keys.
-The naming must be <region_name>.sshkey and <region_name>.gpgkey
+The key files for aikaos are in the path /opt/fiware-aiakos/lib/public/keys.
+The naming must be <region_name>.sshkey and <region_name>.gpgkey (use lowercase is mandatory)
```

Comment:
The key files for aiakos are stored in the folder ...

(capitalisation is mandatory)